edi_exchange_template_oca/migrations/18.0.1.1.0/pre-migrate.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2025 Camptocamp SA (http://www.camptocamp.com)
```

Comment:
@simahawk I need your help here.
1. Do you have an idea how we can test the script?
2. Code review of deprecated feature.
Thanks

Reply:
the version should be 1.2.0, I fix it